src/sentry/feedback/endpoints/feedback_ingest.py

```diff
@@ -0,0 +1,154 @@
+from sentry.api.bases.project import ProjectPermission
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.api.serializers.base import serialize
+from sentry.constants import ObjectStatus
+from sentry.feedback.models import Feedback
+from sentry.feedback.serializers import FeedbackSerializer
+from sentry.models import Organization, ProjectKey
+from sentry.models.project import Project
+from sentry.utils.sdk import bind_organization_context, configure_scope
+
+
+class FeedbackValidator(serializers.Serializer):
+    contexts = serializers.JSONField(required=False)
+    start_timestamp = serializers.FloatField(required=False)
+    tags = serializers.JSONField(required=False)
+    timestamp = serializers.FloatField(required=False)
+    transaction = serializers.CharField(required=False)
+    type = serializers.CharField(required=False)
+    transaction_info = serializers.JSONField(required=False)
+    platform = serializers.CharField(required=False)
+    event_id = serializers.CharField(required=True)
+    environment = serializers.CharField(required=False)
+    release = serializers.CharField(required=False)
+    sdk = serializers.JSONField(required=False)
+    user = serializers.JSONField(required=False)
+    request = serializers.JSONField(required=False)
+
+    def validate(self, data):
+        ret: Dict[str, Any] = {}
+        ret["data"] = {
+            "contexts": data["contexts"],
+            "tags": data["tags"],
+            "transaction": data["transaction"],
+            "type": data["type"],
+            "transaction_info": data["transaction_info"],
+            "platform": data["platform"],
+            "environment": data["environment"],
+            "release": data["release"],
+            "sdk": data["sdk"],
+            "user": data["user"],
+            "request": data["request"],
+        }
+        ret["date_added"] = datetime.datetime.fromtimestamp(data["timestamp"])
+        ret["feedback_id"] = data["event_id"]
+        ret["url"] = ""
+        ret["project_id"] = self.context["project"].id
+        ret["replay_id"] = ""
+        ret["message"] = ""
+
+        return ret
+
+
+class ProjectMonitorPermission(ProjectPermission):
+    scope_map = {
+        "GET": ["project:read", "project:write", "project:admin"],
+        "POST": ["project:read", "project:write", "project:admin"],
+        "PUT": ["project:read", "project:write", "project:admin"],
+        "DELETE": ["project:read", "project:write", "project:admin"],
+    }
+
+
+@region_silo_endpoint
+class FeedbackIngestEndpoint(Endpoint):
+    publish_status = {
+        "GET": ApiPublishStatus.EXPERIMENTAL,
+        "POST": ApiPublishStatus.EXPERIMENTAL,
+    }
+    owner = ApiOwner.FEEDBACK
+
+    authentication_classes = (
+        DSNAuthentication,
+        TokenAuthentication,
+        OrgAuthTokenAuthentication,
+        ApiKeyAuthentication,
+    )
+
+    permission_classes = (ProjectMonitorPermission,)
+
+    def convert_args(
```

Comment:
we'll have to figure out if we want to modularize this w/ the monitor ingest endponts. but i believe those are going to go away, so maybe its best just to keep the copy/pasted code. curious what you think @evanpurkhiser

Reply:
Yeah let's leave it. We are going to remove our monitor endpoints in the future. If you want you can call this out in a comment and mark it with a "This comment should go away when the `MonitorEndpoints` have been removed" or sometihng